### PR TITLE
Added missing info to Wiki section.

### PR DESCRIPTION
**Pull request by n8225**
Added missing licenses and programming languages to Wiki section.

### CODE REVIEW

**Comment by nodiscc** [comment]
Great! Thank you

File: README.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ Some [Groupware](#groupware) solutions also feature calendar/address book editin
     * [Vector](https://matrix.org/blog/project/vector/) - A glossy Matrix web client with an emphasis on performance and usability. ([Source Code](https://github.com/vector-im/vector-web)) `Apache` `Javascript`
     * [Matrix Console Web](https://matrix.org/blog/project/matrix-console-web/) - A web client meant to be a showcase of Matrix capabilities, and reference implementation of the Matrix standard. ([Source Code](https://github.com/matrix-org/matrix-angular-sdk)) `Apache` `Javascript`
   * [Syndie](https://www.syndie.de) - Syndie is a libre system for operating distributed forums. `CC0` `Java`
-  * [TextBelt](https://github.com/typpo/textbelt) - An outgoing SMS API that uses carrier-specific gateways to deliver your text messages for free, and without ads. ` ⚠ ` `MIT` `JavaScript`
+  * [TextBelt](https://github.com/typpo/textbelt) - An outgoing SMS API that uses carrier-specific gateways to deliver your text messages for free, and without ads. `MIT` `JavaScript`  `⚠ `
```
Comment (AndyR207) [inline README.md]: * [TextBelt](https://github.com/typpo/textbelt) ` ⚠ ` - An outgoing SMS API that uses carrier-specific gateways to deliver your text messages for free, and without ads. `MIT` `JavaScript`

Comment (n8225) [inline README.md]: fixed

**Comment by nodiscc** [comment]
Obsoleted by your other PR https://github.com/Kickball/awesome-selfhosted/pull/636.

**Pull request closed without merging**